### PR TITLE
Use std::to_string() instead of std::ostringstream

### PR DESCRIPTION
**Pull request by congwang**
Since C++11 std::to_string() is available to convert numbers to strings.

### CODE REVIEW

File: heron/common/src/cpp/metrics/time-spent-metric.cpp [hunk 1]
```diff
@@ -65,12 +65,11 @@ void TimeSpentMetric::GetAndReset(const sp_string& _prefix,
...
     start_time_ = now;
   }
-  std::ostringstream o;
-  o << total_time_msecs_;
+  std::string val = std::to_string(total_time_msecs_);
   total_time_msecs_ = 0;
   proto::system::MetricDatum* d = _message->add_metrics();
   d->set_name(_prefix);
-  d->set_value(o.str());
+  d->set_value(val);
```
Comment (huijunw) [inline heron/common/src/cpp/metrics/time-spent-metric.cpp]: ? d->set_value(std::to_string(total_time_msecs_));

**Comment by huijunw** [comment]
what c++ version does Heron require?

**Comment by objmagic** [comment]
@huijunw we support C++11

File: heron/common/src/cpp/metrics/time-spent-metric.cpp (hunk 1 above)
Comment (congwang) [inline heron/common/src/cpp/metrics/time-spent-metric.cpp]: We could do that but need to move 'total_time_msecs_ = 0' down.

**Pull request merged**